packages/framework-integration-tests/tsconfig.json

```diff
@@ -14,10 +14,13 @@
     "noFallthroughCasesInSwitch": true,
     "forceConsistentCasingInFileNames": true,
     "experimentalDecorators": true,
-    "emitDecoratorMetadata": true,
     "lib": [
       "ES2019",
-      "ES2020"
+      "ES2020",
+      "DOM"
```

Comment:
Why DOM?

Reply:
Good catch! This is not necessary anymore, so I'll remove it!